### PR TITLE
Use Explicit Flag To Track When Crank Timer Has Elapsed

### PR DESCRIPTION
**Pull request by tholonious**

### CODE REVIEW

**Comment by tholonious** [comment]
Current dependencies on/for this PR:
* feature/proof-of-stake
  * **PR #649** <a href="https://app.graphite.dev/github/pr/deso-protocol/core/649" target="_blank">Graphite</a> 
    * **PR #724** <a href="https://app.graphite.dev/github/pr/deso-protocol/core/724" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/deso-protocol/core/724?utm_source=stack-comment).

File: consensus/types.go [hunk 1]
```diff
@@ -158,7 +158,8 @@ type FastHotStuffEventLoop struct {
 	// Signifies whether or not the event loop has constructed a QC or timeout QC for the current view.
 	// This is an optimization that is useful to prevent the event loop from signaling the server multiple
 	// times for the same view.
```
Comment (lazynina) [inline consensus/types.go:160]: Does this comment apply to both variables? Do we need update it at all?

**Review by lazynina — APPROVED** [review]
Just a little comment nit, otherwise LGTM

File: consensus/types.go (hunk 1 above)
Comment (tholonious) [reply consensus/types.go:160]: Good catch. I need to update this comment to describe both fields now.

**Pull request merged**